attributes/default.rb

```diff
@@ -59,14 +71,20 @@
 # icinga2 resources data bag
 default['icinga2']['databag'] = 'icinga2'
 
-default['icinga2']['run_dir'] = '/var/run/icinga2'
+default['icinga2']['var_dir'] = if node['platform'] == 'windows'
+                                  'C:/ProgramData/icinga2/var'
```

Comment:
Its might not be the best choice but it works for me currently on Server 2012 R2

Maybe a
https://github.com/chef-cookbooks/windows/blob/master/libraries/windows_helper.rb#L32-L34

would be a better choice